stellarphot/settings/models.py

```diff
@@ -435,18 +449,50 @@ class Observatory(BaseModelWithTableRep):
 
     """
 
-    name: str
+    name: Annotated[str, Field(description="Name of the observatory")]
     latitude: Annotated[
-        Latitude, _UnitQuantTypePydanticAnnotation, BeforeValidator(add_degree_to_float)
+        Latitude,
+        _UnitQuantTypePydanticAnnotation,
+        BeforeValidator(add_degree_to_float),
+        Field(
+            description="Latitude of the observatory",
+            examples=[
+                "46.8738",
+                "46.8738 degree",
+                "46:52:25.68 degree",
+                "46d52m25.68s",
+            ],
+        ),
     ]
     longitude: Annotated[
         Longitude,
         _UnitQuantTypePydanticAnnotation,
         BeforeValidator(add_degree_to_float),
+        Field(
+            description="Longitude of the observatory",
+            examples=[
+                "-96.7678",
+                "-96d46m04.08s",
+                "263.2322",
+                "263.2322 degree",
+                "263d13m55.92s",
+            ],
+        ),
+    ]
+    elevation: Annotated[
+        QuantityType,
+        WithPhysicalType("length"),
+        Field(
+            description="Elevation of the observatory",
+            examples=["1000 m", "1 km", "6.685e-9 au"],
```

Comment:
I like the inclusion of AU, but given this is a *stellar* photometry package maybe pc instead. :)